addons/hr_holidays/views/hr_views.xml

```diff
@@ -188,7 +187,7 @@
                         </span>
                     </div>
                 </button>
-                <button name="%(act_hr_employee_holiday_request)d"
+                <button name="%(act_hr_employee_time_off_dashboard)d"
                         type="action"
                         class="oe_stat_button"
                         icon="fa-calendar"
```

Comment:
You can directly call the python method like this (you'll need to move the definition to hr.employee tho):

```suggestion
                <button name="action_time_off_dashboard"
                        type="object"
                        class="oe_stat_button"
                        icon="fa-calendar"
```

Reply:
I tried with type="object" but it didn't work. The button is an action button added to the employee form (hr.employee model) and the method action_time_off_dashboard is in the hr.leave model so the act_hr_employee_time_off_dashboard action is used for translating the models

Reply:
You can move the method `action_time_off_dashboard` to hr.employee :-)

Reply:
Done, thanks a lot :)